src/engraving/rendering/dev/paint.cpp

```diff
@@ -129,21 +131,14 @@ void Paint::paintScore(draw::Painter* painter, Score* score, const IScoreRendere
             }
 
             // Draw page elements
-            bool disableClipping = false;
-
-            if (!painter->hasClipping()) {
-                painter->setClipping(true);
-                painter->setClipRect(pageRect);
```

Comment:
With this change, we can no longer specify a custom clipping area... I'm also wondering how it affects performance, since this code was added to reduce the CPU load during playback (only the playback cursor rect is redrawn)

Reply:
You can still specify a clip rect (namely via opt.frameRect) but with this change it will be intersected with the page rect, to avoid drawing outside page rect.

Reply:
Reversed that change, couldn't get it to avoid that playback cursor repaint issue.